go/enclave/rpc/rpc_utils.go

```diff
@@ -60,20 +60,26 @@ func GetViewingKeyAddressForTransaction(tx *common.L2Tx) (gethcommon.Address, er
 }
 
 // ExtractCallParamTo extracts and parses the `to` field of an eth_call request.
-func ExtractCallParamTo(callParams []byte) (gethcommon.Address, error) {
+func ExtractCallParamTo(callParams []byte) (*gethcommon.Address, error) {
 	var paramsJSONMap []interface{}
 	err := json.Unmarshal(callParams, &paramsJSONMap)
 	if err != nil {
-		return gethcommon.Address{}, fmt.Errorf("could not parse JSON params in eth_call request. JSON params are: %s. Cause: %w", string(callParams), err)
+		return nil, fmt.Errorf("could not parse JSON params in eth_call request. JSON params are: %s. Cause: %w", string(callParams), err)
 	}
 
-	txArgs := paramsJSONMap[0] // The first argument is the transaction arguments, the second the block, the third the state overrides.
```

Comment:
Can you leave this comment in?

Reply:
Good point. Moved it to where it was previously, makes more sense.